src/App.tsx

```diff
@@ -1,26 +1,205 @@
-/* eslint-disable max-len */
+/* eslint-disable react-hooks/rules-of-hooks */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import {
+  addTodo,
+  deleteTodo,
+  getTodos,
+  toggleTodo,
+  USER_ID,
+} from './api/todos';
+import { ErrorMessage, FilterStatus, Todo } from './types/Todo';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { Error } from './components/Error';
 
 export const App: React.FC = () => {
+  const [selectedTodo, setSelectedTodo] = useState<Todo[]>([]);
```

Comment:
it's not selected , it's todos
```suggestion
  const [todos, setSelectedTodo] = useState<Todo[]>([]);
```